_data/simple-icons.json

```diff
@@ -5752,6 +5752,11 @@
             "source": "https://www.geocaching.com/about/logousage.aspx",
             "guidelines": "https://www.geocaching.com/about/logousage.aspx"
         },
+        {
+            "title": "Geode",
+            "hex": "8D7ACF",
```

Comment:
Maybe we can go with a color from their website. They use `#f5dd9f` for every title. The color is quite similar to the first wave we see in the background (`#f4d48e`). Not an easy one!

Reply:
Hello! Lead developer of Geode here - we tend to use the gradient that ranges from purple (`#8d7acf`) to gold (`#f5dd9f`) where-ever possible, but for monochrome having it be fully purple (`#8d7acf`) definitely feels a lot more fitting for Geode than gold :)

Reply:
Thank you for responding! We'll stick with purple then :)